drivers/neopixel/neopixel.py

```diff
@@ -1,44 +1,44 @@
 # NeoPixel driver for MicroPython
 # MIT license; Copyright (c) 2016 Damien P. George, 2021 Jim Mussared
 
-from micropython import const
 from machine import bitstream
 
-_BITSTREAM_TYPE_HIGH_LOW = const(0)
-_TIMING_WS2818_800 = (400, 850, 800, 450)
-_TIMING_WS2818_400 = (800, 1700, 1600, 900)
+# G R B W
+_O = (1, 0, 2, 3)
 
 
 class NeoPixel:
-    ORDER = (1, 0, 2, 3)
```

Comment:
IIRC this is part of the public API, to be able to override the `ORDER` (otherwise there's no way to support LEDs with a different ordering).

See `esp32/modules/apa106.py`.

Reply:
OK, thanks. Updated branch (+7 bytes).